spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  before { get user_posts_path(user_id: User.first.id) }
```

Comment:
- [ ] Previously, all your tests were failing, now half of them are failing. From the error messages, User.first, and Post.first (as used below) do not work because they are `nil`. You have not created them.

  I proposed you modify this before action to the code below
  ```
  before do
    User.create(name: "john doe", bio: "john's bio", posts_counter: 0, photo: "fig-1")
    Post.create(title: "first post", text: "first post text", author_id: User.first.id)
    get user_posts_path(user_id: User.first.id)
  end
  ```
  ![half-tests](https://user-images.githubusercontent.com/54848363/180236240-e61df0e5-d5eb-4e62-8252-5834e5f7cc9b.jpg)

Reply:
Hello @DammyShittu 

While running the test on my pc, all of them are passing successfully now I'm confused and I don't know what change to make when everything is okay on my side. 
Here you can find the screenshot of the tests in my pc:

![Screenshot from 2022-07-21 16-31-46](https://user-images.githubusercontent.com/94958024/180240902-124d753c-3127-470d-a75b-a9a4eac4f11b.png)

Please, let me know if you are free for a quick call so you can easily see what is happening 

Thank you.